### ISSUE TITLE
default API version ignored

### ISSUE THREAD

**Issue by jgeorgeson** [python-gitlab/python-gitlab#435]
The example config file [here](http://python-gitlab.readthedocs.io/en/stable/cli.html#cli) shows that `api_version` can go in the `global` section, so I have configured like this

```
[global]
default = prod
ssl_verify = True
timeout = 5
api_version = 4

[prod]
url = https://git.example.com
private_token = blahblah 

[stage]
url = https://git.stage.example.com
private_token = blahblah
```

Running with `--debug` I see it making v3 calls

```
$ gitlab -o json --debug project get --id 1
send: b'GET /api/v3/projects/1 HTTP/1.1\r\nHost: git.example.com\r\nUser-Agent: python-requests/2.18.4\r\nAccept-Encoding: gzip, deflate\r\nAccept: */*\r\nConnection: keep-alive\r\nPRIVATE-TOKEN: blahblah\r\n\r\n'
reply: 'HTTP/1.1 200 OK\r\n'
DEBUG:urllib3.connectionpool:https://git.example.com:443 "GET /api/v3/projects/1 HTTP/1.1" 200 1542
header: Server header: Date header: Content-Type header: Content-Length header: Connection header: Cache-Control header: Etag header: Vary header: X-Frame-Options header: X-Request-Id header: X-Runtime header: Strict-Transport-Security id: 1
path: scratch
```

Adding an explicit `api_version`  to each server resolves this.

**Comment by gpocentek** [comment]
Hi @jgeorgeson 

Thanks for reporting this issue. The problem has been fixed in commit 29bd813 and will be included in the next release (hopefully this week).

**Comment by jgeorgeson** [comment]
awesome, thanks.
